data/models.json

```diff
@@ -100,5 +100,22 @@
             "recall_score": 0.6746,
             "roc_auc_score": 0.9222
         }
+    },
+    "Tanzania_south_2019": {
+        "params": "",
```

Comment:
Could you train with wandb enabled to allow logging these parameters?

Reply:
I tried adding `--wandb`  parameter, but there is some sort of permission needed.